Umbraco-Uno/Creating-Content/Schedueld-Publishing/index.md

```diff
@@ -0,0 +1,41 @@
+---
+versionFrom: 8.0.0
+---
+
+# Scheduled Publishing
+
+Scheduled publishing lets you set a time and a date for when your page should be published. An example, could be if you have a deadline then you can set the publishing for the deadline. Then work on your page until then and then the site will be automatically published at the time and date of when it was scheduled to.
+
+## How to setup Scheduled publishing
+
+To set up scheduled publishing you will need to follow these steps:
+
+1. Navigate to the page you want to publish at a predefined date and time
+2. At the bottom of this page find the green button that says ***Save and Publish***
+3. Next to the ***Save and Publish*** button there's an extra button with an arrow in
```

Comment:
```suggestion
2. At the bottom of this page select the arrow-button next to the ***Save and Publish*** button - this will reveal a set of additional options
```